Overview.bs

```diff
@@ -91,22 +91,99 @@ Introduction {#intro}
 
 <em>This section is not normative.</em>
 
+Incremental Font Transfer (IFT) is a collection of technologies to improve the latency of remote fonts (or "web fonts") on the web. Without this technology, a browser needs to download every last byte of a font before it can render any characters using that font. IFT allows the browser to download only some of the bytes in the file, thereby decreasing the perceived latency between the time when a browser realizes it needs a font and when the necessary text can be rendered with that font.
+
 The success of WebFonts is unevenly distributed. This specification allows WebFonts to be used where
 slow networks, very large fonts, or complex subsetting requirements currently preclude their use. For
 example, even using WOFF 2 [[WOFF2]], fonts for CJK languages are too large to be practical.
 
+There are two different methods which can be used to incrementally transfer fonts.
+
+Patch Subset {#patch-subset}
+----------------------------
+
+The first method,
+<a href="patch-incxfer">Patch Subset</a>, uses a server which can generate binary updates to an existing font file.
```

Comment:
> The first method, Patch Subset, uses a server which can generate binary updates to an existing font file.

Is this it, a complete description? I am sure there are many more details that we can and should provide about both methods, to help the first-time reader of the spec understand the IFT design and principles. As it's currently presented, all we are saying is that the first method is using a server, and the second method is very simple - hardly an accurate description.

Reply:
as noted in the meeting today I can make a pass at fleshing this out after this PR gets merged.